src/formatDate.js

```diff
@@ -0,0 +1,35 @@
+let formatDate = function(date){
+  date = new Date(date);
+  let month = date.getMonth()+1;
+  let day = date.getDate();
+  let year = date.getFullYear();
+
+  if (month===1){
+    month = "Janurary"
+  } else if (month===2){
+    month = "February"
+  } else if (month===3){
+    month = "March"
+  } else if (month===4){
+    month = "April"
+  } else if (month===5){
+    month = "May"
+  } else if (month===6){
+    month = "June"
+  } else if (month===7){
+    month = "July"
+  } else if (month===8){
+    month = "August"
+  } else if (month===9){
+    month = "September"
+  } else if (month===10){
+    month = "October"
```

Comment:
there's gotta be an easier way to do this! But if it works, it ships :)

Reply:
thats my life motto!